app/screens/Dashboard/CharactersRolls.tsx

```diff
@@ -47,6 +47,8 @@ export default function CharactersRolls (props) {
     onEditCharacter
   } = props
   const list = data || []
+  list.sort((a, b) => (a.initiative < b.initiative) ? 1 : -1)
```

Comment:
This must be done in the State Machine specifically in the api.fake.ts file for the Characters. you don't want business / state logic done in your presentation layer

Reply:
Ok, I'll try again.